read-fonts/src/font_data.rs

```diff
@@ -258,3 +258,12 @@ impl AsRef<[u8]> for FontData<'_> {
         self.bytes
     }
 }
+
+//kind of ugly, but makes FontData work with FontBuilder. If FontBuilder stops using
+//Cow in its API, we can probably get rid of this?
```

Comment:
We should perhaps create, and cite, an issue to not use Cow in the FontBuilder API?

Reply:
#155